packages/resolve-es-file/src/index.js

```diff
@@ -4,16 +4,17 @@ const ENCODING = 'utf8';
 
 const loadEvents = path =>
     new Promise((resolve, reject) =>
-        fs.readFile(
-            path,
-            ENCODING,
-            (err, content) =>
-                (err && err.code !== 'ENOENT'
-                    ? reject(err)
-                    : resolve(content ? JSON.parse(`[${content.replace(/,$/, '')}]`) : []))
-        )
+        fs.readFile(path, ENCODING, (err, content) => {
+            if (err && err.code !== 'ENOENT') {
+                reject(err);
+                return;
+            }
+            resolve(content ? JSON.parse(`[${content.replace(/,$/, '')}]`) : []);
+        })
     );
 
+const compareEvents = (a, b) => a.timestamp - b.timestamp;
```

Comment:
Is it really need to compare events here? Because we always append new events at the end of a file.